autoware_map_msgs/msg/PointCloudCellMetadata.msg

```diff
@@ -0,0 +1,5 @@
+string cell_id
+float32 min_x
+float32 min_y
+float32 max_x
+float32 max_y
```

Comment:
Could be troublesome to change, but how about reusing [PointCloudMapCellWithID.msg](https://github.com/autowarefoundation/autoware_msgs/blob/main/autoware_map_msgs/msg/PointCloudMapCellWithID.msg) (or either it reuses this new message)?

Reply:
I thought so too. :+1: 
I thought so too. I would like to change it once merged as it will affect the existing one.